models/tensorflow/nnf_tf_freezer/nnf_tf_freezer.py

```diff
@@ -15,10 +15,11 @@
 from tensorflow.core.framework.tensor_shape_pb2 import TensorShapeProto
 from tensorflow.tools import graph_transforms
 from typing import List
+from convert_graph_fp16 import* 
 
 class nnf_tf_freezer(object):
     def __init__(self, frozen_graph= "frozen_graph.pb", const_folding=True, run_graph=True, xla=False, parallel=0,
-    warmup=5, num_iter=10, run_const_folded_graph=False, debug=False, is_training=False):
+    warmup=5, num_iter=10, run_const_folded_graph=False, debug=False, is_training=False, is_fp16=False):
```

Comment:
What this param actually do is to covert graph to fp16graph, I think to_fp16 is more accurate.

Reply:
Sure, I will fix this.